src/tabs/osd.html

```diff
@@ -151,7 +151,8 @@ <h1 class="tab_title">
             </div>
 
             <!-- Font Manager dialog -->
-            <div id="fontmanagercontent" style="display:none; width:720px;">
+            <dialog closedby="any" id="fontmanagerdialog" class="html-dialog" style="width: 750px;">
+            <div class="html-dialog-content" style="height: 430px; overflow-y: scroll;">
```

Comment:
Perhaps
```suggestion
            <div class="html-dialog-content" style="height: 530px; overflow-y: scroll;">
```

![image](https://github.com/user-attachments/assets/46839124-3427-4cda-8aee-cd3c9a43ef06)